examples/generative/md/vq_vae.md

```diff
@@ -795,10 +795,10 @@ them to our decoder to generate novel images.
 ```python
 # Create a mini sampler model.
 inputs = layers.Input(shape=pixel_cnn.input_shape[1:])
-x = pixel_cnn(inputs, training=False)
-dist = tfp.distributions.Categorical(logits=x)
-sampled = dist.sample()
-sampler = keras.Model(inputs, sampled)
+layers = pixel_cnn(inputs, training=False)
```

Comment:
This is not a layer or set of layers, but a tensor. We should name it something that reflects this, e.g. `outputs`